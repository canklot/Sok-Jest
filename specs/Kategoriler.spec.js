import TestComponent from '../components/Kategoriler.vue'
import { mount } from '@vue/test-utils'

test('mount a vue component', () => {
    const wrapper = mount(TestComponent
    )
    expect(wrapper).toMatchSnapshot()
  })