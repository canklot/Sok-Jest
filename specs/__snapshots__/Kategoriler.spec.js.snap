// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount a vue component 1`] = `
<div class="Kategoriler">
  <div class="Baslik">Süt &amp; süt ürünleri</div> <button class="Kategori">Süt</button> <button class="Kategori">Yoğurt</button> <button class="Kategori">Ayran &amp; kefir</button> <button class="Kategori">Beyaz Peynir</button> <button class="Kategori">Krem Peynir</button> <button class="Kategori">Yöresel Gurme Peynir</button> <button class="Kategori">Kaymak &amp; Krema</button> <button class="Kategori">Tereyağ &amp; Margarin</button> <button class="Kategori">Maya</button> <button class="Kategori">Kaşar Peyniri</button> <button class="Kategori">Kaşar Peyniri</button> <button class="Kategori">Kaşar Peyniri</button>
</div>
`;
